generate-moz-grammars/generate-mozcpp.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# This script updates the mozcpp grammar automatically.
+#
+# Usage: ./generate-moz-grammars/generate-mozcpp.sh
+
+# FIXME we need to remove this line once we are going to use
+# the tree-sitter-cpp bindings
+# Update tree-sitter-cpp submodule such that is the same one used within the
+# tree-sitter-mozcpp directory
+git submodule update --remote tree-sitter-cpp
+
+# Enter the mozcpp directory
+pushd tree-sitter-mozcpp
+
+# Download tree-sitter-cpp
+git clone https://github.com/tree-sitter/tree-sitter-cpp.git
+
+# Download tree-sitter-c inside the tree-sitter-cpp repository
+pushd tree-sitter-cpp
+
+# Clone tree-sitter-c repository
+git clone https://github.com/tree-sitter/tree-sitter-c
+
+# Update the path to the tree-sitter-c repository contained in
+# the grammar.js file
+NO_FIRST_LINE=$(sed '1d' grammar.js)
+echo "const C = require(\"./tree-sitter-c/grammar\")$NO_FIRST_LINE" > grammar.js
+
+# Exit tree-sitter-cpp directory
+popd
+
+# Init npm
+npm init -y
+
+# Install a small module that lets the parser be used from Node
+npm install --save nan
+
+# Install the Tree-sitter CLI
+npm install --save-dev tree-sitter-cli
+
+# Generate moz-cpp grammar
+./node_modules/.bin/tree-sitter generate
+
+# Delete node_modules
+rm -rf node_modules
+
+# Delete package files
+rm -rf package-lock.json package.json
+
+# Delete tree-sitter-cpp
+rm -rf tree-sitter-cpp
```

Comment:
We should remove the `tree-sitter-c` directory too, right?

Reply:
Since `tree-sitter-c` is inside `tree-sitter-cpp`, we remove both of them in one fell swoop